tardis/model.py

```diff
@@ -241,3 +242,11 @@ def update_plasmas(self, initialize_nlte=False):
                                                                'macroatom'):
             self.transition_probabilities = (
                 self.plasma_array.transition_probabilities)
+  
+    def postprocess(self):
+        self.Edotlu_norm_factor = (1 / 
+            (self.time_of_simulation * self.tardis_config.structure.volumes))
+        exptau = 1 - np.exp(- 
+                            self.runner.line_lists_tau_sobolevs.reshape(-1,
+                                self.tardis_config.structure["velocity"]["num"]) ) 
+        self.Edotlu = self.Edotlu_norm_factor*exptau*self.Edotlu_estimators
```

Comment:
I wouldn't do this post-processing here. As I've said, the 1- exp(-tau) factor should already be included in the estimator increments in the C part.